app/controllers/standard_steps_controller.rb

```diff
@@ -18,6 +21,14 @@ def update
 
   private
 
+  def after_successful_update
+
```

Comment:
Can you put a `#noop` comment in here to just clarify it's emptiness? 🕳

Reply:
Yep!